WiserTaskScheduler/AutoUpdater/Models/SlackSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace AutoUpdater.Slack.modules;
+
+public class SlackSettings
+{ 
+    /// <summary>
+    /// Gets or sets the name of the SlackChannel.
+    /// </summary>
+    public  string Channel { get; set; }
```

Comment:
I'm not going to reject the PR for this, but please reduce the amount of spaces between `public` and `string` to a single space.